src/bp/core/config/botpress.config.ts

```diff
@@ -198,3 +199,27 @@ export interface AuthStrategyLdap {
 }
 
 export type FieldMapping = { [key in keyof Partial<AuthUser>]?: string }
+
+export interface MonitoringConfig {
+  /**
+   * To enable server monitoring, you need to enable the Pro version and configure your Redis server.
+   * @default false
+   */
+  enabled: boolean
+  /**
+   * The interval between data collection of metrics and usage. The lower the value brings more details,
+   * but comes at the cost of more storage required & processing time when viewing data.
+   * @default 10s
+   */
+  collectionInterval: string
+  /**
+   * Data older than this will be cleared periodically.
```

Comment:
can you add some info on the timeformat used or point to the lib we use. I know it seems clear that 10d is 10 days but for I'd prefer to add a link to the lib then having to answer question about this because someone entered a non valid interval format

Reply:
I'll add something in documentation, and i've added details on top of the botpress confgi file about that